documentation/airport-guides/ksav.md

```diff
@@ -0,0 +1,25 @@
+# KSAV- Savannah International Airport
+
+## About SAV
+Savannah/Hilton Head International Airport (IATA:SAV, ICAO: KSAV, FAA LID: SAV) is a commercial and military-use airport in Savannah, Georgia. Savannah/Hilton Head International provides travelers with access to Savannah, Georgia, and Hilton Head Island, South Carolina, as well as neighboring areas including Bluffton and Beaufort, South Carolina and the Golden Isles region of Coastal Georgia.
+
+Owned by the City of Savannah and managed by the Savannah Airport Commission, Savannah/Hilton Head International is located seven nautical miles (8 mi, 13 km) northwest of the Savannah Historic District. The airport's passenger terminal is directly accessible to Interstate 95 between Savannah and the suburban city of Pooler. Its previous names include Savannah International Airport, Travis Field and Chatham Field.
+
+## Airspace
+The SAV TRACON airspace covers about 40nm from the Savannah Airport and from the surface to 10,000 feet.
+
+## Procedures
+Savannah has no SIDS or STARS, so all aircraft enter and exit the airspace through ATA’s and DTA’s. All aircraft will exit or enter the airspace through a fix listed below. All arrivals will fly to their entry fix first and then proceed direct to the Savannah VOR (SAV). All departure will proceed direct to their first fix. 
```

Comment:
No apostrophe needed in ATAs and DTAs. The 's' marks the plural, not the possessive 😉 

> All departure will ...
missing 's' 😅

Reply:
Fixed